content/en/smart-contracts/programmatic-storage/Interfaces/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "RaaS base and full interfaces"
```

Comment:
Note: base and full interface naming is being removed. See more in thread here: https://filecoinproject.slack.com/archives/C0370V3JTPB/p1693420691727189?thread_ts=1693411733.707419&cid=C0370V3JTPB

Base interface is now called "self-hosted RaaS". 
Full interface is now called "aggregator hosted RaaS"